SocialFilter/sections/general_elemhide.txt

```diff
@@ -59,6 +65,57 @@
 ##.p-article__articleWrapper ~ div.p-article__share
 ! hide share button in youtube player
 ##.ytp-share-button
+! X (formerly Twitter)
+~twitter.com,~x.com##[title="Compartir en Twitter"]
+~twitter.com,~x.com##body .twitter-share-button
+~twitter.com,~x.com##input.tweet
+~twitter.com,~x.com##[onclick^="window.open('https://x.com/intent/tweet"]
+~twitter.com,~x.com##a[onclick^="window.open('https://x.com/share"]
+~twitter.com,~x.com##[onclick^="window.open('https://twitter.com/intent/tweet"]
+~twitter.com,~x.com##a[onclick^="window.open('https://twitter.com/share"]
+##a[onclick^="window.open('http://x.com/share"]
+~twitter.com,~x.com##a[onclick^="window.open ('http://www.twitter.com/share?url="]
+~twitter.com,~x.com##a[onclick*="window.open"][onclick*="'https://twitter.com/intent/tweet?"]
+~twitter.com,~x.com##a[onclick^="window.open('http://twitter.com/home?status"]
+##a[onclick^="window.open('http://twitter.com/share"]
+~twitter.com,~x.com##a[href^="https://www.x.com/share"]
+~twitter.com,~x.com##a[href^="https://www.twitter.com/share"]
+~twitter.com,~x.com##a[href^="https://x.com/home/?"]
+~twitter.com,~x.com##a[href^="https://x.com/home?"]
+~twitter.com,~x.com##a[href^="https://x.com/intent/retweet?tweet_id"]
+~twitter.com,~x.com##a[href^="https://twitter.com/home/?"]
+~twitter.com,~x.com##a[href^="https://twitter.com/home?"]
+~twitter.com,~x.com##a[href^="https://twitter.com/intent/retweet?tweet_id"]
+~twitter.com,~x.com##a[href^="http://www.x.com/home?"]
```

Comment:
`http://` can be used without `~twitter.com,~x.com` because they use only HTTPS on the x.com / twitter.com

Reply:
What website uses `##a[href^="http://www.x.com/home?"]`? https://publicwww.com/websites/%22http%3A%2F%2Fwww.x.com%2Fhome%3F%22/

Reply:
@Yuki2718 fixed.

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/180173#discussion_r1615797811

Reply:
> `http://` can be used without `~twitter.com,~x.com` because they use only HTTPS on the x.com / twitter.com

Agreed